src/mount/fuse/mount_config.cc

```diff
@@ -221,6 +222,9 @@ void usage(const char *progname) {
 "    -o enablefilelocks=0|1      enables/disables global file locking "
 				"(disabled by default)\n"
 "    -o nonempty                 allow mounts over non-empty file/dir\n"
+"    -o sfsignoreflush           DANGEROUS. Ignore flush usual behavior by "
+				"replying SUCCESS to it inmediately. Targets fast creation of "
+				"small files, but may cause data loss during crashes.\n"
```

Comment:
Same here.

Reply:
yes lets remove DANGEROUS here....

Reply:
@dmga44 typo in word: ~inmediately~ - should be: immediately

Reply:
Done, both.